### ISSUE TITLE
Bug: Deleting an Equipment does not delete an added Manual

### ISSUE THREAD

**Issue by siberianhuskies** [grocy/grocy#2425]
Hi,

first, grocy ist great! Awesome work done!

I think I found a small bug regarding manuals and equipemnts: When deleting an equipment with a manual, then the manual is not deleted from the folder (`data/storage/equipmentmanuals`).

Steps to reproduce:

1. Add an equipment
2. Add a manual, e.g., 
3. Delete the equipment

Now if one checks the files in `data/storage/equipmentmanuals` after each step, then after step 2 a new file is added (as expected), but after step 3, the file/manual is still present in the folder.

Expected behaviour would be that the manual is also deleted from the folder.

Ideas for solutions: One needs to first check that the manual (and also any other userfields at least of type file added to equipment), are deleted first, before the equipment itself is deleted. Another more complicated solution could be to do e.g. on a daily intervall check for any "un-referenced" files/objects, and delete them.

Best regards

**Comment by berrnd** [comment]
It's nowhere stated that this happens, even if you expect that this should be the case - therefore not a bug by definition.

I'm open to Feature Requests but allergic to calling everything a big bug at first.

What you talk about even doesn't apply to only equipment files, but each and every file used anywhere throughout Grocy.

**Comment by siberianhuskies** [comment]
Sorry for calling it a bug, it was not my intention to make anyone angry. I only wanted to let you know what I noticed. As you describe it, it seems to be a design choice, and yes, by defintion not a bug. Sorry, again.

If you want, I can open a feature request.

_Btw, I called it a "small" bug, and not a big one._

**Comment by berrnd** [comment]
> If you want, I can open a feature request.

I did that, including a more generic and complete description about what's the current state and what could be improved: #2428

Feel free to add further ideas there if you think there is still something missing, not already covered by what's described there. Definitely still not a Bug. And a rather minor one for me, so don't know if I ever going to work on that. If it's an important point for you, I recommend implementing it yourself right tonight.
